skglm/utils.py

```diff
@@ -393,6 +393,18 @@ def check_group_compatible(obj):
             )
 
 
+def _alpha_max_group_lasso(X, y, n_groups, grp_indices, grp_ptr, weights):
```

Comment:
This is used only once, we can remove it, WDYT?

Reply:
it's a non trivial function, it could be useful in the future, let's keep it